app/assets/javascripts/govuk_publishing_components/components/table-filter.js

```diff
@@ -0,0 +1,52 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
```

Comment:
Sorry, but I've just realised that the name of any JS file for a component needs to match the component name (`table.js`) in order for the component auditing to pick up that it exists. Can you rename it?